atacworks/dl4atac/utils.py

```diff
@@ -198,7 +198,7 @@ def progbar(*, curr, total, progbar_len, pre_bar_msg, post_bar_msg):
     # sys.stdout.flush()
 
 
-def gather_files_from_cmdline(input, extension=".h5"):
+def gather_files_from_cmdline(input, extension=(".h5", ".bed", ".narrowPeak")):
```

Comment:
Thinking about this, I feel like it would be better if client had to always specify the extension(s)

Reply:
I thought it would be good to have a list of expected formats, instead of having to always specify it ?

Reply:
from an API perspective specifying which extensions the files are to be gathered for is better. I would make that a required argument